adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -772,7 +860,7 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
     // convert avro to sequence dictionary
     val rgd = loadAvroReadGroupMetadata(filePath)
 
-    AlignedReadRDD(rdd, sd, rgd)
+    AlignedReadRDD(rdd, sd, rgd, partitionMap = extractPartitionMap(filePath))
```

Comment:
@fnothaft @heuermh I want to standardize these apply calls to the different GenomicRDDs in the PR; some of them have an empty line before and some do not. Preferences?